server/app/controllers/applicant/UpsellController.java

```diff
@@ -213,7 +213,8 @@ public CompletionStage<Result> download(
                   pdfExporterService.generateApplicationPdf(
                       applicationMaybe.join().get(),
                       /* showEligibilityText= */ false,
-                      /* includeHiddenBlocks= */ false);
+                      /* includeHiddenBlocks= */ false,
```

Comment:
Does include includeHiddenBlocks true only for admins? If yes, let us fold them too into into isAdmin